### PR TITLE
EJB should stand for Enterprise Jakarta Beans

### PR DESCRIPTION
**Pull request by hendrikebbers**
As far as I understand the abbreviation 'EJB' should stand for Enterprise Jakarta Beans since this release

### CODE REVIEW

**Comment by arjantijms** [comment]
> As far as I understand the abbreviation 'EJB' should stand for Enterprise Jakarta Beans since this release

That's not correct. We're no longer allowed to use EJB (with some exceptions), and it therefor has not changed meaning.

**Comment by hendrikebbers** [comment]
ok, but than 'Enterprise Jakarta Beans' needs to be added to chapter 1.3, right?

**Comment by hendrikebbers** [comment]
Some for 2.8: 'The EJB Timer Service, defined by the Jakarta Enterprise Beans specification.' <- Maybe EJB should be removed here.

**Comment by hendrikebbers** [comment]
From my point of view it would be better to remove the term 'EJB' completely from the doc. It is used only 5 times in the text. Here 'Enterprise Java Beans' or 'Enterprise Jakarta Beans' can be named directly. I assume it will mostly be 'Enterprise Jakarta Beans'.

**Comment by hendrikebbers** [comment]
Question: Is it correct that you add the trademark sign correctly for 'Jakarta™ Enterprise Beans' to but 'Enterprise JavaBeansTM'?

**Comment by hendrikebbers** [comment]
@arjantijms ^^several comments do not belong to this PR but I just have a look at some specs and try to do some reviews :) Do not know where to add my comments...

**Comment by arjantijms** [comment]
> ok, but than 'Enterprise Jakarta Beans' needs to be added to chapter 1.3, right?

Yes, indeed. That's a few lingering remnants that seem to have been missed before. In some occasions we can refer to EJB if it's clearly and explained that we're referring to Oracle's trademark, but in most cases its simplest to just replace it.

**Comment by hendrikebbers** [comment]
ok, will create a new PR once I have the 'eclipsefdn/eca' check working...

**Comment by tkburroughs** [comment]
Note, there is no such thing as "Enterprise Jakarta Beans".  When referring to the new spec, it should be "Jakarta Enterprise Beans", or when referring to the technology, it should be "enterprise beans".  I would recommend replacing "EJB Timer Service" with "enterprise beans timer service".

File: spec/src/main/asciidoc/1_overview.adoc [hunk 1]
```diff
@@ -41,7 +41,7 @@ The change log for the current version is found in <<change_log>>.
 
 The Jakarta EE Platform specification requires
 support for interceptors. The use of interceptors defined by means of
-the `Interceptors` annotation is required to be supported for EJB and
+the `Interceptors` annotation is required to be supported for Enterprise Jakarta Beans (EJB) and
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/1_overview.adoc:44]: +1 to @tkburroughs , this should be Jakarta Enterprise Beans!

**Pull request closed without merging**